src/ferc_xbrl_extractor/cli.py

```diff
@@ -99,11 +106,38 @@ def parse():
     return parser.parse_args()
 
 
+def load_extracted(
+    extracted: xbrl.ExtractOutput,
+    sqlite_uri: str,
+    duckdb_path: str | None,
+    clobber: bool,
+) -> None:
+    """Write extracted data to SQLite/Duckdb databases."""
+    engine = create_engine(sqlite_uri)
+    if clobber:
```

Comment:
If we decide to keep `clobber` around we should probably clobber the duckdb path too - or make that `CREATE TABLE` conditionally become `CREATE TABLE OR REPLACE`.

Reply:
Decided to just remove the `clobber` option. No need for extra complexity we're not making use of.